src/pretix/base/models/event.py

```diff
@@ -235,7 +235,7 @@ def waiting_list_active(self):
         if not self.settings.waiting_list_enabled:
             return False
         if self.settings.waiting_list_auto_disable:
-            return self.settings.waiting_list_auto_disable.datetime(self) > now()
+            return self.settings.waiting_list_auto_disable.datetime(self) > time_machine_now()
```

Comment:
I just wanted do note it explicitly: waiting_list_auto_disable also influences quota availability (see base/services/quotas.py), where do *not* want the time machine, as that could lead to wrong contents in the quota cache or overbookings. So it's all good as it is, just don't want anyone to add it there as well because it might look like a good idea on first sight :)